segmentstore/server/src/main/java/io/pravega/segmentstore/server/writer/WriterConfig.java

```diff
@@ -52,6 +53,12 @@
     @Getter
     private final Duration flushThresholdTime;
 
+    /**
+     * The minimum number of attributes that should accumulate before flushing them into the Attribute Index.
```

Comment:
It seems odd to configure a minimum in terms of a count without also providing some sort of bound. Like maybe based on time. Otherwise it could sit unflushed indefinitely.

Reply:
There already is a time bound (see line 54 above). It's the same setting used for the regular segment transfer; I decided I didn't want another setting that would have essentially the same value.